source/wasm/build.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Copyright (c) 2020 The Khronos Group Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+NUM_CORES=$(nproc)
```

Comment:
I'm a little concerned that this is not quite right for the cloud.  But I don't have data to back up that concern.
Presumably a cloud instance will have an appropriate limit.

Reply:
I think this 'should just work'™, as it should just se the virtual CPU's of the VM it's running in, but I'll do some experimentation to make sure.

Reply:
I added build output showing number of detected cores. For GitHub actions it shows 2, which AFAIK matches the published spec for the GitHub action runners.